public/docs/_examples/style-guide/ts/01-01/app/mock-heroes.ts

```diff
@@ -0,0 +1,11 @@
+// #docregion
+/**
+ * Mock heroes until we can get real data
+ */
+import { Hero } from './hero.model';
+
+export const HEROES: Hero[] = [
+   {id: 1, name: 'Bombasto'},
+   {id: 2, name: 'Tornado'},
+   {id: 3, name: 'Magneta'},
```

Comment:
Please dont worry about these for now. @Foxandxss will catch these once we have @mgechev 's linter in there. Or he can just use an `.editorconfig` to make sure they are all 2 chars